### DISCUSSION TITLE
Github Code Search and API Search through Curl Giving different results

### DISCUSSION THREAD

**Discussion by ghost** [community/community#53660] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

I am attempting an code search from github and it gives me the below result.
![1](https://user-images.githubusercontent.com/131009528/234199847-8cfab42a-ee17-4938-a8d9-f672f350381b.png)
When I am attempting it using the github API, it is a different result. 
![2](https://user-images.githubusercontent.com/131009528/234199898-39233236-fc71-476a-ab3a-dec374b39e05.png)
Is it something I am doing wrong or does anybody know why is this happening.

**Comment by kcrane3576** [comment]
I am also dealing with unexpected results when working with Search API (beta)
- UI
	- attempt: query to get specific files from specific repo
		- path:**/{file_name}+repo:{username}/{repo.name}
	- result:This repository's code is being indexed right now. Try again in a few minutes.

- Search API 
	- response: {"total_count":0,"incomplete_results":false,"items":[]}'
	- status code: 200


I don't want to take away from original post, but seems relevant.

Questions:
1. Will 'Search' also be monitored here: https://www.githubstatus.com/
2. If Github indexing, I thought the response was supposed to be something like below:
	- Status: 202 Accepted
	- Message: Your query is being indexed. Please try again later.

**Comment by ghost** [comment]
I not really sure why this is happening, it was working fine until last Monday. Suddenly I started observing variations in the results fetched by the API. I have not seen anybody mentioning such an issue.

**Reply by kcrane3576** [reply]
@dsoggg You may want to mark this as a bug. Ref: https://github.com/orgs/community/discussions/53799
